NARF tutorial code not compiling

http://www.pointclouds.org/documentation/tutorials/narf_feature_extraction.php

PCL trunk
```bash
Linking CXX executable range_image
CMakeFiles/range_image.dir/range_image.cpp.o: In function `bool pcl::visualization::PCLVisualizer::addPointCloud<pcl::PointWithRange>(pcl::PointCloud<pcl::PointWithRange>::ConstPtr const&, pcl::visualization::PointCloudColorHandler<pcl::PointWithRange> const&, std::string const&, int)':
range_image.cpp:(.text._ZN3pcl13visualization13PCLVisualizer13addPointCloudINS_14PointWithRangeEEEbRKNS_10PointCloudIT_E8ConstPtrERKNS0_22PointCloudColorHandlerIS5_EERKSsi[_ZN3pcl13visualization13PCLVisualizer13addPointCloudINS_14PointWithRangeEEEbRKNS_10PointCloudIT_E8ConstPtrERKNS0_22PointCloudColorHandlerIS5_EERKSsi]+0x2fd): undefined reference to `pcl::visualization::PointCloudGeometryHandlerXYZ<pcl::PointWithRange>::PointCloudGeometryHandlerXYZ(boost::shared_ptr<pcl::PointCloud<pcl::PointWithRange> const> const&)'
CMakeFiles/range_image.dir/range_image.cpp.o:(.rodata._ZTVN3pcl13visualization28PointCloudGeometryHandlerXYZINS_14PointWithRangeEEE[_ZTVN3pcl13visualization28PointCloudGeometryHandlerXYZINS_14PointWithRangeEEE]+0x30): undefined reference to `pcl::visualization::PointCloudGeometryHandlerXYZ<pcl::PointWithRange>::getGeometry(vtkSmartPointer<vtkPoints>&) const'
collect2: error: ld returned 1 exit status
make[2]: *** [range_image] Error 1
make[1]: *** [CMakeFiles/range_image.dir/all] Error 2
make: *** [all] Error 2

```

--- a/visualization/src/point_cloud_handlers.cpp
+++ b/visualization/src/point_cloud_handlers.cpp
@@ -805,7 +805,7 @@ pcl::visualization::PointCloudGeometryHandlerCustom<pcl::PCLPointCloud2>::PointC
 
 // Instantiations of specific point types
 #ifdef PCL_ONLY_CORE_POINT_TYPES
-  PCL_INSTANTIATE(PointCloudGeometryHandlerXYZ, (pcl::PointSurfel)(pcl::PointXYZ)(pcl::PointXYZL)(pcl::PointXYZI)(pcl::PointXYZRGB)(pcl::PointXYZRGBA)(pcl::PointNormal)(pcl::PointXYZRGBNormal)(pcl::PointXYZRGBL))
+  PCL_INSTANTIATE(PointCloudGeometryHandlerXYZ, (pcl::PointSurfel)(pcl::PointXYZ)(pcl::PointXYZL)(pcl::PointXYZI)(pcl::PointXYZRGB)(pcl::PointXYZRGBA)(pcl::PointNormal)(pcl::PointXYZRGBNormal)(pcl::PointXYZRGBL)(pcl::PointWithRange))
   PCL_INSTANTIATE(PointCloudGeometryHandlerSurfaceNormal, (pcl::Normal)(pcl::PointNormal)(pcl::PointXYZRGBNormal))
 #else
   PCL_INSTANTIATE(PointCloudGeometryHandlerXYZ, PCL_XYZ_POINT_TYPES)

